mne/viz/_brain/_brain.py

```diff
@@ -1785,6 +1802,21 @@ def _cortex_colormap(self, cortex):
                             )
         return colormap_map[cortex]
 
+    def _remove(self, item):
+        """Remove actors from the rendered scene."""
+        if item in self._actors:
+            for actor in self._actors[item]:
+                self._renderer.plotter.remove_actor(actor)
```

Comment:
there can be multiple views/scenes, does this work when that's the case?

Reply:
Yes, it's a list of actors for each type

Reply:
I'm more concerned that the `.plotter` is never changed to any other subplot, I'm a bit surprised it works. But as long as you've at least tested by eye / manually that everything works in this case, I'm happy

Reply:
This works, can confirm